Cython/Utility/CythonFunction.c

```diff
@@ -1608,51 +1608,67 @@ static int __pyx_FusedFunction_init(PyObject *module) {
 //////////////////// ClassMethod.proto ////////////////////
 
 #include "descrobject.h"
-CYTHON_UNUSED static PyObject* __Pyx_Method_ClassMethod(PyObject *method); /*proto*/
+static CYTHON_INLINE int __Pyx_PyMethodDescr_Check(PyObject *method);  /*proto*/
+static PyTypeObject *__Pyx_PyMethodDescr_Type(PyObject *method);  /*proto*/
+CYTHON_UNUSED static PyObject *__Pyx_Method_ClassMethod(PyObject *method);  /*proto*/
 
 //////////////////// ClassMethod ////////////////////
 
-static PyObject* __Pyx_Method_ClassMethod(PyObject *method) {
+static CYTHON_INLINE int __Pyx_PyMethodDescr_Check_old(PyObject *method) {
```

Comment:
Not sure why we're keeping this (since it's unused here and not as good as the full version I think)

Reply:
Right, it's unused. Just remove it.